python/tvm/topi/image/resize.py

```diff
@@ -23,6 +23,22 @@
 from .. import tag
 
 
+def can_multiple2div(image, target):
```

Comment:
The names seem to be not so clear. Can they be named as below?
`can_multiple2div` -> `can_convert_multiply_to_intdiv`
`image` -> `origin_size`
`target` -> `scaled_size`